src/main/java/uk/gov/hmcts/dts/fact/model/admin/CourtType.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.dts.fact.model.admin;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonPropertyOrder({ "id","name", "code"})
+public class CourtType {
+
+
+    @JsonProperty("id")
+    private Integer id;
+    @JsonProperty("name")
+    private String name;
+    @JsonProperty("code")
+    private Integer code;
```

Comment:
All the JsonProperty annotations here are redundant. You only need them when you want a field name in the Json string to be different to the field name in the model class.

Reply:
Removed all Json property annotations